components/footer.component.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
```

Comment:
Folder refactor proposal:

common-app/components

We will have the following root folders:

common --> reusable assets that are not domain dependant
common-app --> reusable assets that belogin to the app (domain dependant)
core --> cross cutting assets